pkg/controller/postgresqluser/postgresqluser_controller.go

```diff
@@ -40,6 +40,8 @@ func init() {
 	FlagSet.String("aws-access-key-id", "", "AWS access key id to use for credentials")
 	FlagSet.String("aws-secret-access-key", "", "AWS secret access key to use for credentials")
 	FlagSet.StringToString("host-credentials-user", nil, "Host and credential pairs in the form hostname=user:password. Use comma separated pairs for multiple hosts")
+	FlagSet.Bool("all-databases-enabled-read", false, "Enable usage of allDatabases field in read access requests")
+	FlagSet.Bool("all-databases-enabled-write", false, "Enable usage of allDatabases field in write access requests")
```

Comment:
Let's ensure that this is never true for a production environment.

Reply:
👍